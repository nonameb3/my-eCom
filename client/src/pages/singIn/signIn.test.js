import React from "react";
import { shallow } from "enzyme";

import SignIn from "./signIn.component";

it("should rander signin component", () => {
  expect(shallow(<SignIn />)).toMatchSnapshot();
});
