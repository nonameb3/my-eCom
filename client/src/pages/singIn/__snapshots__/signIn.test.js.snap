// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should rander signin component 1`] = `ShallowWrapper {}`;
